.travis.yml

```diff
@@ -58,6 +58,20 @@ matrix:
           packages:
             - g++-8
 
+      # ARM g++ 6 on Linux with C++14
```

Comment:
Specialized configuration is not needed.

Reply:
yeah, I will revert this commit, was not sure that this warning can also be produced on x86_64 gcc-6 (tested it on Compile Explorer)

Reply:
I don't think we need another CI config for this.

Reply:
reverted